src/averageAges.js

```diff
@@ -15,11 +15,16 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const menArray = century
+    ? people
+      .filter(person => person.sex === 'm')
+      .filter(person => century === Math.ceil(person.died / 100))
```

Comment:
these filters can be combines, just write the condition inside of one filter (with && operator)